js/app2.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+class Book {
+  #list
+
+  constructor() {
+    const localList = JSON.parse(localStorage.getItem('stuff'));
+    if (localList !== null) {
+      this.#list = localList.arr;
+    }
+  }
+
+  #g(j) {
+    return this.document.getElementById(j);
+  }
+
+  #stack(templateString) {
+    const template = this.document.createElement('template');
+    template.innerHTML = templateString.trim();
+    document.getElementById('books').appendChild(template.content.firstChild);
+  }
+
+  updateList() {
+    document.getElementById('books').innerHTML = '';
+    for (let j = 0; j < this.#list.length; j++) {
+      const book = this.#list[j];
+      this.#stack(`<div id="${book.title}">
+        <p>${book.title}</p>
+        <p>${book.author}</p>
+        <button id="${j + book.title}" value="${j}" onclick="target(this)">Remove</button>
+      </div>
+      <hr>`);
+    }
+  }
+
+  updateLocalStorage() {
+    localStorage.setItem('stuff', JSON.stringify({
+      arr: this.#list,
+    }));
+  }
+
+  addBook() {
+    const title = this.#g('t').value;
+    const author = this.#g('a').value;
+    const book = {};
+    book.author = author;
+    book.title = title;
+    this.#list.push(book);
+    this.updateLocalStorage();
+    this.updateList();
+    this.#g('t').value = '';
+    this.#g('a').value = '';
+  }
```

Comment:
* Currently when i try to add a book i get errors in the console please fix it 🙏 
![Screenshot (152)](https://user-images.githubusercontent.com/52497246/124920492-47d12a00-dfef-11eb-90a8-25c48248f0ed.png)